plugins/zfunctions/functions/funced

```diff
@@ -51,7 +51,7 @@ eos
 fi
 
 # open the function file
-if [[ -z "$VISUAL" ]]; then
+if [[ ! -z "$VISUAL" ]]; then
```

Comment:
This should be `-n`, not `! -z`. Can you change that? If so, I will merge.

Reply:
TIL; thanks, updated.